src/bootstrap-table.js

```diff
@@ -1894,6 +1894,10 @@
         this.$tableBody.find('tbody tr:first-child:not(.no-records-found) > td').each(function (i) {
             $footerTd.eq(i).outerWidth($(this).outerWidth());
         });
+        if (this.$tableFooter.find('table').width() != this.$tableBody.find('table').innerWidth())
```

Comment:
Well as I tested a little bit, my fix is not good at all. It solves the problem for my case in Bootstrap 2, but it creates the same problem for Bootstrap 3. So I think it should be better not to use my fix and find a better solution. Maybe I should create an issue for this bug?